pint/timing_utils/timing_setup.py

```diff
@@ -0,0 +1,32 @@
+# # Licensed under the  BSD 3-clause license - see LICENSE
+#
+#
+# """ timing_setup.py defines a set of functions for setting up PINT for timing.
+# """
+#
+# from .toa import get_TOAs
+# from .models import get_model
+#
+#
+#
+# def check_components(model, required_components=['astrometry', 'spindown']):
+#     """Check if a timing model has sufficent components for high precision
+#     pulsar timing.
```

Comment:
Better to explain what this does and doesn't do. For example, is a binary model required? What about a DM model? Anything to do with the Solar System? I can do high-precision timing with nothing but a spin-down model if I'm using already-barycentered X-ray data on an isolated pulsar. What about noise components?